app/js/voice.js

```
@@ -37,32 +37,35 @@
                 const hasKelion = t.includes('kelion') || t.includes('chelion');
                 const hasKira = t.includes('kira') || t.includes('chira');
-                const hasK = t === 'k' || t.startsWith('k ');
-
-                if ((hasKelion || hasKira || hasK) && event.results[i].isFinal) {
+                if (hasKelion || hasKira) {
                     const targetAvatar = hasKira ? 'kira' : 'kelion';
-                    const currentAvatar = window.KAvatar.getCurrentAvatar();
-                    if (targetAvatar !== currentAvatar) {
+                    if (window.KAvatar && targetAvatar !== window.KAvatar.getCurrentAvatar()) {
                         window.KAvatar.loadAvatar(targetAvatar);
                         document.querySelectorAll('.avatar-pill').forEach(b => b.classList.toggle('active', b.dataset.avatar === targetAvatar));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)